shelley/chain-and-ledger/formal-spec/crypto-primitives.tex

```diff
@@ -67,18 +67,21 @@ \section{Cryptographic primitives}
   \emph{Constraints}
   \begin{align*}
     & \forall (sk, vk) \in \KeyPair,~ d \in \Data,~ \sigma \in \Sig \cdot
-    \sign{sk}{d} = \sigma \implies \verify{vk}{d}{\sigma}
+    \sign{sk}{d} = \sigma \implies (vk, d, \sigma) \in \verify
   \end{align*}
   \emph{Notation for serialized and verified data}
   \begin{align*}
-    & \serialised{x} & \text{serialised representation of } x\\
+    & \serialised{x} & \text{serialised representation of } x (NSF: restrict the type of x?)\\
     & \mathcal{V}_{\var{vk}}{\serialised{d}}_{\sigma} = \verify{vk}{d}{\sigma}
     & \text{shorthand notation for } \fun{verify}
   \end{align*}
   \caption{Cryptographic definitions}
   \label{fig:crypto-defs-shelley}
 \end{figure}
 
+(NSF: what is $\Data$? Does stand for something like $\exists x. Serialisable x
```

Comment:
@JaredCorduan  is there a link to a document where `\Data` is specified?

Reply:
It's in appendix E of the delegation design doc: https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/delegationDesignSpec/latest/download-by-type/doc-pdf/delegation_design_spec

Reply:
I changed the `\Data` type to `\Ser` in this file because `\Data` is a specific type (it has 5 constructors and is a safe type for putting information inside blocks, like metadata for example). `\Ser` I used to denote the serialized representation of anything. This is what gets signed.